### DISCUSSION TITLE
PvP Loot chest screenshot

### DISCUSSION THREAD

**Discussion by Joel21241** [runelite/runelite#15335] [Ideas]
Is there a way someone can either implement the choice to have loot chest screenshots on the normal screenshot plugin, dont know if its hard to do.

**Comment by Nightfirecat** [comment]
The screenshot plugin already captures screenshots of most kinds of chest loot. What isn't it picking up?

**Reply by Joel21241** [reply]
The pvp loot keys chest

**Reply by Nightfirecat** [reply]
Mostly the issue is naming them. Those screenshots wouldn't really be all that useful right now because they can't be associated with the name of the player who dropped them.
